docs/whatsnew/5.1.rst

```diff
@@ -118,6 +119,26 @@ least-squares optimization algorithm from scipy. These new fitters are:
   via the `scipy.optimize.least_squares` function.
 * :class:`~astropy.modeling.fitting.DogBoxLSQFitter`, which uses the dogleg algorithm.
 
+.. _whatsnew-iers-handling:
+
+Allow time conversions without predictive Earth rotation data (IERS-A)
+======================================================================
+
+Some time conversions like UTC -> UT1 require additional Earth rotation data for
+full accuracy. These data are provided by the online IERS service as the IERS-A
+tables and are downloaded as required. In some use cases this download is not
+desired or possible. Examples include an application where full accuracy is not
+required, running on a cluster, or the rare instances when the IERS server and
+mirror are not available. For these cases there is a new config item
+`iers.conf.iers_degraded_accuracy <astropy.utils.iers.Conf.iers_degraded_accuracy>` which specifies the behavior when times are
```

Comment:
Nitpick: formatting would be nicer with consistent line length (also for changelog item)

Reply:
Not my fault exactly, that came from accepting review commit suggestions.

Reply:
I'll fix since there will be at least another commit.